core/core.go

```diff
@@ -566,6 +556,29 @@ func (t *TextileNode) IsSignedIn() (bool, error) {
 	return err == nil, nil
 }
 
+func (t *TextileNode) JoinThread(mnemonic string, name string) error {
```

Comment:
Could we call this CreateAlbumByName or something like that? "join" clashes with the method "JoinRoom" and everything is still called albums in the code

Reply:
ah, nvm, CreateAlbum already takes mnemonic and name...

Reply:
Seems like this "setting the public room" is a pretty beta-y / mobile concept, we could just move the logic that says "overwrite it if exists" to mobile package in a very explicit method: SetPublicThread / Album

Reply:
just thinking that we'll soon want a real "create thead" method that should not overwrite (so users can create their own threads), what you think?